specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/capacityReservation.json

```diff
@@ -873,6 +879,12 @@
           "type": "string",
           "description": "A unique id generated and assigned to the capacity reservation by the platform which does not change throughout the lifetime of the resource."
         },
+        "platformFaultDomainCount": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "This is the readonly property that is populated and set in the response to specify the Maximum Fault Domain Count that capacity reservation support for the VM Size.<br><br>Minimum api-version: 2022-08-01."
```

Comment:
No need to mention this is a read only property it is already specified via the readOnly attribute set on property in the spec, let us update the comment to "Specifies the value of fault domain count that Capacity Reservation supports for requested VM Size. <br>NOTE:  The fault domain count of a resource that supports Capacity Reservation must less than or equal to this value.